common/src/main/java/com/microsoft/identity/common/crypto/AndroidWrappedKeyLoader.java

```diff
@@ -99,9 +99,6 @@ public class AndroidWrappedKeyLoader extends AES256KeyLoader {
     private final CachedData<SecretKey> mKeyCache = new CachedData<SecretKey>() {
         @Override
         public SecretKey getData() {
```

Comment:
Is the override needed anymore here?

Reply:
The class is marked as abstract, so I think this is still needed.

Reply:
At least something to make this a new class is instantiatable.  I guess this method in particular is not required.

Reply:
I learned something about Java here -- even without the @Override, the braces make this an anonymous class.  And since none of the individual methods are abstract, this is enough to be constructible.